pages/linux/rtcwake.md

```diff
@@ -0,0 +1,36 @@
+# rtcwake
+
+> Enter a system sleep state until specified wakeup time relative to your bios clock.
+> Commands need to be run as superuser since shutdowns are involved.
+
+- Check your hardware clock info and exit:
+
+`rtcwake -m show -v`
+
+- Show if an allarm is set:
+
+`rtcwake -m show`
+
+- Disable a previous set alarm:
+
+`rtc -m disable`
+
+- Suspend to ram and wakeup after 10 seconds:
+
+`# rtcwake -m mem -s {{10}}`
+
+- Suspend to disk _(higher power saving)_ and wakeup 10 minutes later:
+
+`# rtcwake -m disk --date +{{15}}m`
+
+- Suspend to disk and wakeup 1 hour later so play some music with mpv:
```

Comment:
Maybe ....`so play some music with mpv` would read better as ....`and play some music with mpv`?

Reply:
I was also unsure about the use of `mpv`.. It's a cool combination but maybe "and run a command" would be better because not every system has `mpv` installed.. i.e. they could have `vlc`